### DISCUSSION TITLE
Dataset shows invalid in Datastudio after March 2022

### DISCUSSION THREAD

**Discussion by flatform** [owid/covid-19-data#2613] [Q&A]
Hello,

Having problems renewing the Covid-19 Dataset in Google Datastudio. **Last successful refresh of the dataset was in 08/03/2022** (March 08, 2022). We are getting the error message below:

**"File Upload Error Details
There was an error in processing your upload.

This file has processing errors.
Invalid column header name(s) [all the header names are shown here in the original error message] in uploaded data."**

After examining the file, we saw that  total_tests was in the wrong column so we did the following:

**moved total_tests after new_tests**

**after that the file opens, but still produces errors in many if not all graphs** (still testing).

if the changes you made were not intentional, please revert to the old column formatting/structure: new_tests in Z column, total_tests in AA column, as well other changes that we may have not found.

thank you in advance.

**Comment by lucasrodes** [comment]
Hi @flatform,
We reordered columns to be a bit more consistent in their order.

We strongly recommend accessing columns by their header name and not column index.

**Reply by flatform** [reply]
Yes, we assumed you did it for consistency reasons.

When we access columns we do it by header name, though on importing (to refresh the database) we have no control over the conversion. Only new columns are accepted in google datastudio, when placed to the far right? Re-arranging brakes things up we guess, or not? We cannot be sure.

Till March 08 2022, we had no issues. Did you make any column re-arrangements prior to that?

Just trying to see where the problem is, because even after we manually change column order, .csv is accepted yet charts brake.

Did you also change format of the data?

Regards

**Reply by lucasrodes** [reply]
Hi @flatform,
I am not familiar with how Google Data Studio operates, so I may not be able to fully assist you.

Also, I am not sure I know which file you refer to. From your wording, I assume that you are using the excel file https://covid.ourworldindata.org/data/owid-covid-data.xlsx, is that right?

We have sometimes changed its format, adding / modifying columns, you can review all changes here: https://github.com/owid/covid-19-data/tree/master/public/data#changelog

**Reply by flatform** [reply]
Yes, we are referring to the complete dataset you provided in your reply, in .csv format.

We have rebuilt the whole project from scratch to make it work again.

Since as you said header names are important and not columns, may we suggest you avoid column swapping in the future, unless it is absolutely necessary.

After 2 years of stable data, it was at least strange to see changes. 

Thank you for replying, keep up the good work.

**Reply by lucasrodes** [reply]
Noted down, thanks for reporting and sorry for the inconveniences!
